tests/platform_tests/test_advanced_reboot.py

```diff
@@ -63,14 +68,28 @@ def test_fast_reboot_from_other_vendor(duthosts,  rand_one_dut_hostname, request
     advancedReboot.runRebootTestcase()
 
 @pytest.mark.device_type('vs')
-def test_warm_reboot(request, get_advanced_reboot, verify_dut_health,
+def test_warm_reboot(request, get_advanced_reboot, verify_dut_health, tbinfo,
     advanceboot_loganalyzer, capture_interface_counters):
     '''
     Warm reboot test case is run using advacned reboot test fixture
 
     @param request: Spytest commandline argument
     @param get_advanced_reboot: advanced reboot test fixture
     '''
+    if "dualtor" in tbinfo["topo"]["name"]:
```

Comment:
All other tests in this script are  also valid for dualtor. Hence, this logic should be in common place (new fixture?)

Reply:
Added single_tor/dual_tor fixture